rclcpp/test/rclcpp/executors/test_reinitialized_timers.cpp

```diff
@@ -32,7 +32,7 @@ class TestTimersLifecycle : public testing::Test
   void TearDown() override {rclcpp::shutdown();}
 };
 
-TYPED_TEST_SUITE(TestTimersLifecycle, ExecutorTypes);
+TYPED_TEST_SUITE(TestTimersLifecycle, ExecutorTypes, );
```

Comment:
why this change? is it a typo?

Reply:
no, it's another clang warning

Reply:
It's specifically for: https://ci.ros2.org/view/nightly/job/nightly_linux_clang_libcxx/1986/gcc/folder.-1649941887/fileName.-411323968/

> must specify at least one argument for '...' parameter of variadic macro [-Wgnu-zero-variadic-macro-arguments] 35 | TYPED_TEST_SUITE(TestTimersLifecycle, ExecutorTypes); | ^

adding a comma (and nothing afterwards) does not change the behaviour and satisfies clang.

Reply:
I agree with @fujitatomoya , the correct approach is to pass `ExecutorTypeNames` there.